### ISSUE TITLE
Firewall rules were not implemented with script

### ISSUE THREAD

**Issue by Bucky-McDonuts** [StarshipEngineer/OpenVPN-Setup#13]
Hello,
I had to manually add the line for opening up the firewall on the RPi with the rule that was created by your script.  I added the following line to /etc/network/interfaces after the line 'iface eth0 inet manual' and it solved my issue of not connecting to the internet through the VPN.

pre-up /etc/firewall-openvpn-rules.sh

I apologize if this doesn't make sense.  This is my first issue submitted through github.

Thanks,
Bucky

**Comment by StarshipEngineer** [comment]
Did you set up your Pi with a static IP address before running the script? If so, was it entered manually or through your router's dhcp reservations?

The script is written to add the reference to the firewall rules (the line you quoted) in /etc/network/interfaces immediately following the static IP info entered into that file, and assumes you've already entered the static IP info manually. It might seem a bit picky, but assigning a static IP that way can be done pretty easily on any Pi, while not all users might know how or be able to use dhcp reservations on their router.

**Comment by Bucky-McDonuts** [comment]
I made the assignment on my router before starting up the pi for the first time.

**Comment by StarshipEngineer** [comment]
In that case the script isn't making an error, it's just not written to accommodate that method of establishing a static IP. I'm new at shell scripting, and don't know how to handle both cases with the same script. For the sake of generality and minimizing how much users need to mess around with their router configuration, I went with manually setting a static IP on the Pi, as mentioned. Manually entering the pre-up in interfaces like you did does the job just fine though (some of the sources I used for reference do that).
